### PR TITLE
Audit _shame.scss

### PR DESCRIPTION
**Pull request by ataker**
## Description
Complete an audit of `_shame.scss`. Remove unused code and add comments to code remaining about its use.

Closes [Audit _shame.scss](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2044)

## Testing done
Almost none, only removed code that had no use in `vets-website` or `content-build`. Some I ran to double check an edge case, but mostly untested.

## Acceptance criteria
- [ ] Unused code is removed

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jamigibbs — COMMENTED** [review]
- Is this bit only for IE support? If so, we might be able to remove it too since we no longer support that browser: 
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/04bd7ceda89c0984a5fec3aa56b57a69b32bf5e6/packages/formation/sass/_shame.scss#L189-L192

- And if we're creating a release after this, package.json version will need to be updated.

File: packages/formation/sass/_shame.scss [hunk 1]
```diff
@@ -3,77 +3,58 @@
...
-
-#content .section {
-  fieldset {
-    h4 {
-      padding: 0;
-    }
-  }
-}
 
-.edit-checkbox {
```
Comment (micahchiang) [inline packages/formation/sass/_shame.scss:61]: I see one usage of `edit-checkbox` in vets-website [here](https://github.com/department-of-veterans-affairs/vets-website/blob/804fc87411ee25b96f1424ac26f1473fb2624daf/src/applications/facility-locator/sass/facility-locator.scss#L363-L365), but it's only styling `label:visited`. Do we know if this instance relies on these styles for the states below?

**Review by micahchiang — APPROVED** [review]
looks good, thanks for taking care of this @ataker! I just left one comment about the `edit-checkbox` styles.

File: packages/formation/sass/_shame.scss [hunk 2]
```diff
@@ -3,77 +3,58 @@
...
-
 body {
+  // used a lot in vets-website
   .row {
     max-width: 62.5em;
   }
 
   // We have no container element, so elements that are full width have to redefine the container layout every time
   // if they want to line up.
+  // unused in content-library
```
Comment (Andrew565) [inline packages/formation/sass/_shame.scss:14]: Should all these references to content-library say content-build?

File: packages/formation/sass/_shame.scss (hunk 1 above)
Comment (ataker) [reply packages/formation/sass/_shame.scss:61]: I can't find any actual uses of `edit-checkbox` in the markup, so we should be safe. [Double checked](https://dsva.slack.com/archives/C0FQSS30V/p1700082344533909?thread_ts=1700080546.822219&cid=C0FQSS30V) with someone working on the facilities team.

File: packages/formation/sass/_shame.scss (hunk 2 above)
Comment (ataker) [reply packages/formation/sass/_shame.scss:14]: Great catch!

File: packages/formation/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/formation",
-  "version": "7.1.0-color-beta",
+  "version": "8.0.0",
```
Comment (ataker) [inline packages/formation/package.json:3]: Bumped major version since removing could be a breaking change. Let me know if you disagree

**Review by jamigibbs — APPROVED** [review]

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**